src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure implements FindArea {
+    private int firsLeg;
```

Comment:
There is a typo in the variable name `firsLeg`. It should be corrected to `firstLeg` for consistency and clarity.